taxonomy/api/v1/views.py

```diff
@@ -1,18 +1,22 @@
 """
 Taxonomy API views.
 """
+from django_filters.rest_framework import DjangoFilterBackend
 from rest_framework import permissions
 from rest_framework.mixins import ListModelMixin, RetrieveModelMixin
 from rest_framework.viewsets import GenericViewSet, ModelViewSet
 
-from django_filters.rest_framework import DjangoFilterBackend
 from django.db.models import Prefetch
 
+from taxonomy.api.filters import SkillNameFilter
+from taxonomy.api.permissions import IsOwner
```

Comment:
I do not see any use of `IsOwner` in the PR, please remove if this import is not being used anywhere.

Reply:
it is being used [here](https://github.com/openedx/taxonomy-connector/blob/e3af609230fc6776ade4fa216bf44d674ccabec2/taxonomy/api/v1/views.py#L86)
I just isorted the file